packages/quantic/force-app/main/default/lwc/quanticCaseAssistInterface/quanticCaseAssistInterface.js

```diff
@@ -62,7 +62,7 @@ export default class QuanticCaseAssistInterface extends LightningElement {
                     analytics: {
                       analyticsMode: 'legacy',
                       ...(document.referrer && {
-                        originLevel3: document.referrer,
+                        originLevel3: document.referrer.substring(0, 256),
```

Comment:
Please add this as well to the QuanticRecommendations interface as well as I think we are not setting originLevel3 on this component.